src/core/AnimatedTransform.js

```diff
@@ -12,11 +12,6 @@ function sanitizeTransform(inputTransform) {
           property: key,
           nodeID: value.__nodeID,
         });
-      } else {
```

Comment:
There's no need to do anything with Animated.Nodes if layout props with primitive types have been changed. It will imply that new props will be attached and component will be re-rendered. The ref to view (stored in Animated.Props) will be updated